### DISCUSSION TITLE
MCDU Display is moving

### DISCUSSION THREAD

**Discussion by EisernUnion** [flybywiresim/aircraft#5774] [Help]
Hi,
didnt find anything about the following issue.
When toggle the camera by klicking the middle mouse button inside the cockpit and toggling back... the MCDU Display is movable and its getting out of it's "centered position" .... to get a better understand what i mean ... take a look at a short clip....

https://youtu.be/t44J1pmqpYs

**Comment by tracernz** [comment]
That is a new one!
